docs/docs/docs/developer-resources/testing/getting-started.md

```diff
@@ -0,0 +1,96 @@
+---
+id: getting-started
+title: Testing - Getting Started
+slug: /developer-resources/testing/getting-started
+sidebar_position: 16
```

Comment:
Take a look at the Docusaurus site. Are these relative positions OK? For example, the current order because of the sequence used is:

1. Getting Started
2. Platform Tests
3. Setup Guide